rpmconf/Upgrade/zmupgrade.pm

```diff
@@ -215,8 +215,8 @@ sub upgrade {
        $isLdapMaster = 0;
   }
   my ($startBuild,$targetBuild);
-  ($startVersion,$startBuild) = $startVersion =~ /(\d\.\d\.\d+_[^_]*)_(\d+)/;
-  ($targetVersion,$targetBuild) = $targetVersion =~ m/(\d\.\d\.\d+_[^_]*)_(\d+)/;
+  ($startVersion,$startBuild) = $startVersion =~ /(\d+\.\d\.\d+_[^_]*)_(\d+)/;
```

Comment:
should we also support versions like 10.11.12 ? I think middle 11 will not match with regex